diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -138,12 +141,31 @@ fun ASTNode.isFollowedByNewline() =
             it.treeNext.elementType == WHITE_SPACE && it.treeNext.text.contains("\n")
         } ?: false
 
+/**
+ * This function is similar to isFollowedByNewline(), but there may be a comment after the node
+ */
+fun ASTNode.isFollowedByNewlineWithComment() =
+        parent({ it.treeNext != null && it.treeNext.treeNext != null }, strict = false)?.let {
+            it.treeNext.treeNext.elementType == EOL_COMMENT && it.treeNext.treeNext.isFollowedByNewline()
+        } ?: false ||
+                parent({ it.treeNext != null }, strict = false)?.let {
+                    it.treeNext.run {
+                        when (elementType) {
+                            WHITE_SPACE -> text.contains("\n")
+                            EOL_COMMENT -> isFollowedByNewline()
```

Comment:
```suggestion
                            EOL_COMMENT, BLOCK_COMMENT, KDOC -> isFollowedByNewline()
```
All other types of comments can occupy a single line, so we should cover cases like `foo() /* comment */` too